Makefile

```diff
@@ -131,9 +131,15 @@ rpm: $(TARFILE) $(SPEC)
 	rm -r "`pwd`/rpmbuild"
 	rm -r "`pwd`/output" "`pwd`/build"
 
+NETWORK=--no-network
+# fedora can be built offline; the rest does not pull podman containers in image-create yet
+ifneq ($(filter fedora,$(TEST_OS)),)
```

Comment:
oops, `filter` is wrong here; TEST_OS is not a list of strings, but a single string, and you want substring match. You probably want `find`; see https://www.gnu.org/software/make/manual/html_node/Text-Functions.html